dags/process_elife_zip_dag.py

```diff
@@ -294,9 +294,11 @@ def send_article_to_content_service(upstream_task_id: str = None, **context) ->
 
     try:
         response.raise_for_status()
-    except HTTPError as e:
-        logger.error('%s HTTP Error= %s', response.status_code, response.text)
-        raise e
+    except HTTPError:
+        response_xml = etree.parse(BytesIO(response.content))
```

Comment:
what happens if the XML is not parseable? In an imperfect world happens more often than we think, especially with gateway in the middle :sweat_smile: would still try to log the raw `response.content` string in that case

Reply:
If the response is not parsable then that would indicate that the schema for errors is not consistent and I would expect the system to fall over.

Reply:
I agree it should fall over, just wanted a way to know what the response was when it fell over, since it can be difficult to find out what happened.